frontend/tailwind.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
                     light: '#12302e',
                     dark: '#081615',
                 },
+                orange: {
+                    500: '#f97316',
```

Comment:
You can set this to `DEFAULT` which will allow us to use `text-orange` and still add future versions when we need it! (500 is also always the default)